src/main/kotlin/omok/model/BoardCoordinate.kt

```diff
@@ -0,0 +1,15 @@
+package omok.model
+
+data class BoardCoordinate(private val number: Int) {
```

Comment:
```suggestion
data class BoardCoordinate(val number: Int) {
```
로 선언된다면 getNumber 함수가 굳이 없어도 될 것 같아요!

Reply:
넵!

Reply:
저는 Data클래스라고 할지라도. 직접 변수에 접근하기보다, get함수를 이용하는데, 이렇게 해도 될까요?